data-node/src/test/java/org/graylog/datanode/filesystem/index/IndicesDirectoryParserTest.java

```diff
@@ -46,15 +48,15 @@ void testOpensearch2() throws URISyntaxException {
         Assertions.assertThat(result.nodes())
                 .hasSize(1)
                 .allSatisfy(node -> {
-                    Assertions.assertThat(node.nodeVersion()).isEqualTo("2.10.0");
+                    Assertions.assertThat(node.nodeVersion()).isEqualTo(OPENSEARCH_VERSION_IN_DATANODE);
```

Comment:
The index directory has been created by 2.10 and will stay in that version if we won't regenerate the files with a newer one. This is why the test is now failing here.

Reply:
reverted, thx